js/blog.js

```diff
@@ -0,0 +1,61 @@
+     'blogP2':'Nunc vitae libero laoreet, elementum lacus sed, ultricies nibh. Nam elementum commodo erat, lobortis iaculis justo elementum eget. Mauris id lacus massa. In maximus elit euligula cursus consequat. Nullam a massa eu mi fringilla ornare sodales non augue. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a dolor ut nulla mattis eleifend id a metus. In porta orci nec pellentesque ullamcorper. Phasellus sodales ipsum et lacus eleifend, nec sodales lacus aliquet.'
+    },
+
+    {'blogImg':"images/blog-2.jpg" , 
+    'blogAltText':'Multiple dogs on leash outside',
+    'blogHeader':'How to Walk Multiple Dogs',
+    'blogP1':'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque blandit lectus non nisl feugia tristique. Cras massa purus, pharetra at diam vel, tincidunt volutpat metus. Cras sed rutrum dolor, id maximus felis. Nullam pulvinar ex non lectus tempus, id euismod lectus ullamcorper. Sed eleifend augue et libero faucibus semper. Curabitur dapibus enim elit, in molestie nisl euismod vel. In efficitur lacinia lorem, suscipit blandit est volutpat nec.',
+    'blogP2':'Nunc vitae libero laoreet, elementum lacus sed, ultricies nibh. Nam elementum commodo erat, lobortis iaculis justo elementum eget. Mauris id lacus massa. In maximus elit euligula cursus consequat. Nullam a massa eu mi fringilla ornare sodales non augue. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a dolor ut nulla mattis eleifend id a metus. In porta orci nec pellentesque ullamcorper. Phasellus sodales ipsum et lacus eleifend, nec sodales lacus aliquet.'
+   },
+
+   {'blogImg':"images/blog-3.jpg" , 
+   'blogAltText':'Woman playing ball with her dog at sunset',
+   'blogHeader':'Teach Your Dog to Fetch',
+   'blogP1':'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque blandit lectus non nisl feugia tristique. Cras massa purus, pharetra at diam vel, tincidunt volutpat metus. Cras sed rutrum dolor, id maximus felis. Nullam pulvinar ex non lectus tempus, id euismod lectus ullamcorper. Sed eleifend augue et libero faucibus semper. Curabitur dapibus enim elit, in molestie nisl euismod vel. In efficitur lacinia lorem, suscipit blandit est volutpat nec.',
+   'blogP2':'Nunc vitae libero laoreet, elementum lacus sed, ultricies nibh. Nam elementum commodo erat, lobortis iaculis justo elementum eget. Mauris id lacus massa. In maximus elit euligula cursus consequat. Nullam a massa eu mi fringilla ornare sodales non augue. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a dolor ut nulla mattis eleifend id a metus. In porta orci nec pellentesque ullamcorper. Phasellus sodales ipsum et lacus eleifend, nec sodales lacus aliquet.'
+  }
+];
+
+
+for (let i = 0; i<blogContent.length; i++) {
+    
+// each blog is assigned to an aritcle which are enclosed in a container
```

Comment:
Nice work on the dynamically generated posts. This looks great! Parsing through data and generating HTML elements is used everywhere in web. You'll do some more of this in the next course as well with vue.